.travis.yml

```diff
@@ -16,10 +16,22 @@ virtualenv:
   system_site_packages: true
 install:
   - pip install -r https-everywhere-checker/requirements.txt
-before_script:
-  - sh -e /etc/init.d/xvfb start
+  - pip install -r test/chromium/requirements.txt
 env:
   - DISPLAY=':99.0'
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - wget https://chromedriver.storage.googleapis.com/2.18/chromedriver_linux64.zip
+  - unzip chromedriver_linux64.zip
+  - sudo mv chromedriver /usr/bin/chromedriver
+  - sudo apt-get install -y python-software-properties
```

Comment:
So yea...

https://travis-ci.org/EFForg/https-everywhere/builds/78606378, "Chrome version must be >= 43.0.2357.0"